src/main/java/seedu/duke/command/edit/EditTaskCommand.java

```diff
@@ -13,8 +13,8 @@
 
     private int taskID;
     private String newTaskDescription;
-    public static final String COMMAND_WORD = Parser.COMMAND_WORD_EDIT + "-t";
-    public static final String FORMAT = Parser.COMMAND_WORD_EDIT + "-t" + " <task ID> <new task description>";
+    public static final String COMMAND_WORD = "edit";
```

Comment:
Same concern that COMMAND_WORD should exist in the parent class EditCommand instead